README.md

```diff
@@ -1,2 +1,161 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [To Do List Project] <a name="about-project"></a>
+
+
+**[Personal Portfolio Website Project]** is a php project that you allowed to manage your daily tasks. You can add or remove tasks. Make them completed. It's fully dynamic one age app.
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+This project only works client side right now.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://microverse.org/">HTML&CSS&JavaScrit</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Add & Remove Tasks]**
+- **[Fully Responsive]**
+- **[Very Light Project Env]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Coming Soon...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :movie_camera: Demonstration Video <a name="demonstration"></a>
+
+Coming Soon...
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Operating System
+Web Browser
+
+### Setup
+
+Clone this repository to your desired folder.
+Open html file.
+
+
+### Install
+
+There is no need to any installation
+
+
+### Usage
+
+To run the project, execute the following command:
+
+Clone this repository and run index.html file
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+### Deployment
+
+You can deploy this project using:
+
+Any text editor.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@s2jack](https://github.com/s2jack)
+- Twitter: [@1stGuler](https://twitter.com/1stGuler)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/selcuk-guler)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Add tags for tasks]**
+- [ ] **[Add inner descriptions]**
+- [ ] **[Monthly and weekly view options]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please drop a comment.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse Family and all my coding partners.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
```

Comment:
- [ ]  While you have made good effort to include the license, unfortunately am not able to open the license using the path provided. can adjust the path to the license. here is a suggestion on how you can solve this.
 ```suggestion
This project is [MIT](./LICENSE) licensed.
```